content/blog/is-dns-migration-a-tricky-affair.md

```diff
@@ -0,0 +1,106 @@
+---
+
+title: "Is DNS migration a tricky affair?"
+
+date: 2022-12-15
+
+draft: false
+
+featured: true
+
+weight: 1
+
+---
+
+Never trust a friend who says DNS migration is an easy affair. Your instinct says it's easy, well don't trust your instinct. In this post, I will be sharing my experience with one such encounter with DNS migration from GoDaddy to Route53.
+
+Before starting, if you are here because of the catchy title and have no idea of what DNS is, or you are one of my many (zero) followers or need to brush up your knowledge on DNS, please refer to these resources:
+1. [What is DNS - Cloudflare](https://www.cloudflare.com/learning/dns/what-is-dns/)
+2. [Deep Dive on DNS - IETF](https://www.youtube.com/watch?v=DV0q9s94RL8)
+
+I won't keep you waiting any longer - I will start my story and explain the reason for my introductory lines.
+# Need for DNS migration
+One of our clients wanted us to move their workload from docker swarm to Amazon ECS. Earlier, they were using HaProxy to load-balance the traffic. This setup also included generation and installation of SSL certificates using in-house scripts, which the client needed to maintain. So, we decided to go with a managed solution to reduce the management overhead and chose AWS ALB and ACM combination. In order for ACM to auto-renew expiring SSL certificates, we decided to use Route53 as the DNS service, ditching GoDaddy's DNS service. BTW, if you are curious, the domain was registered with GoDaddy and hence their DNS service.
```

Comment:
s/docker swarm/Docker Swam/g

Reply:
Maybe we can reword. 

s/So, we decided to go with a managed solution to reduce the management overhead and chose AWS ALB and ACM combination/Since we were moving to ECS, ALB and ACM felt like an obvious choice/g

Reply:
Fixed in 15acecd